MapboxMobileEvents/MMECommonEventData.m

```diff
@@ -77,7 +78,7 @@ + (NSString *)applicationState {
     if (NSBundle.mme_isExtension) {
         return MMEApplicationStateExtension;
     } else {
-        switch (UIApplication.sharedApplication.applicationState) {
+        switch (MMEUIApplicationWrapper.new.applicationState) {
```

Comment:
We will eventually need an application state for 'Extension' to put into location events here, so we can track telemetry coming from extensions: https://github.com/mapbox/trace-maker/issues/97#issuecomment-638330549

Reply:
Yes, that is the line just above.

Because this is leveraging an existing interface
`@property(nonatomic, readonly) UIApplicationState applicationState;`

It was easier to do the inspection above with an isExtension check. Is there an approach you would prefer otherwise?